app/pages/Settings/index.js

```diff
@@ -50,37 +51,49 @@ export default class Settings extends Component {
 
   render() {
     return (
-      <ContentArea title="Settings" noPadding>
-        <div className="settings__label">Network</div>
-        <div className="settings__dropdown">
-          <NetworkPicker />
-        </div>
+      <ContentArea noPadding>
+        <div className="settings__supersection">Wallet Actions</div>
+        <hr className="settings__separator"/>
         <ul className="settings__links">
           <li>
             <a href="#" onClick={this.props.lockWallet}>
-              Log out
+              Lock wallet and logout
             </a>
           </li>
+        </ul>
+        <ul className="settings__links">
           <li>
-            <div className="settings__link" onClick={this.onDownload}>
-              Download logs
-            </div>
+            <Link to="/settings/zap-txs">Delete unconfirmed transactions</Link>
           </li>
         </ul>
-        <div className="settings__section-head">Your recovery seed phrase</div>
         <ul className="settings__links">
           <li>
-            <Link to="/settings/reveal-seed">Reveal</Link>
+            <Link to="/settings/reveal-seed">Reveal recovery seed phrase</Link>
           </li>
         </ul>
-        <div className="settings__section-head">Reset your account</div>
         <ul className="settings__links">
           <li>
-            <Link to="/settings/new-wallet">Create a new wallet</Link>
+            <Link to="/settings/new-wallet">Remove wallet and create new wallet</Link>
           </li>
         </ul>
+        <p />
+        <div className="settings__supersection">Advanced</div>
+        <hr className="settings__separator"/>
         <div className="settings__section-head">HSD API Key</div>
         <input type="text" className="settings__copy-api-key" value={this.props.apiKey} readOnly />
+        <p />
+        <div className="settings__section-head">Network</div>
+        <div className="settings__dropdown">
+          <NetworkPicker />
+        </div>
+        <div className="settings__section-head">Debug</div>
+        <ul className="settings__links">
+          <li>
+            <div className="settings__link" onClick={this.onDownload}>
```

Comment:
Mind removing this link? It doesn't do anything.

Reply:
I have a temporary fix for this: https://github.com/kyokan/bob-wallet/pull/118/commits/42b8190f2aa5f9876ef77924732ecd732b51d813

It will still need future work but we can do that in another PR. At least with this fix it will work sometimes (until the log file gets too big).

Or, if you like, yeah we can just take it out. This patch is at least a little better than nothing.